modules/ROOT/pages/index-self-managed.adoc

```diff
@@ -107,3 +107,13 @@ For Titanium customers, Runtime Fabric on Self-Managed Kubernetes supports loggi
 == See Also
 
 * xref:install-self-managed.adoc[Install Runtime Fabric on Self-Managed Kubernetes]
+
+[TIP]
+--
+The following tutorials, written by the MuleSoft developer relations team, explain how to provision clusters on EKS, AKS, or GKE
+for installing Runtime Fabric on Self-Managed Kubernetes. These tutorials for provisioning clusters are intended as examples only and are not officially supported by MuleSoft.
```

Comment:
The key here is to let customers know we have an example ingress controller here. So maybe i will say something like "The following tutorials, written by the MuleSoft developer relations team, explain how to provision clusters on EKS, AKS, or GKEfor installing Runtime Fabric on Self-Managed Kubernetes, with Nginx ingress controller as an example".

Reply:
added